### PR TITLE
Separate privacy and terms page

### LINKED ISSUE #2212: Separate privacy and terms page
**Actual Behaviour**
Currently the privacy and terms page are the same.

**Expected Behaviour**
Have separate privacy and terms page.
**Screenshots of the issue**
Screenshot 2019-06-06 at 4 23 23 PM
Screenshot 2019-06-06 at 4 23 15 PM

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2212 
Changes: Added content for Privacy page similar to Google's privacy page.(https://policies.google.com/privacy?hl=en-US)

Demo Link: https://pr-2251-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-06-10 at 11 34 35 PM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@mariobehling @akshatnitd @fragm3 Please review the content of privacy page.

**Comment by harsh-jindal** [comment]
@fragm3 Fixed it

File: src/components/Privacy/Privacy.react.js [hunk 1]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
+        <H1>
+          When you use our services, you’re trusting us with your information.
+          We understand this is a big responsibility and work hard to protect
+          your information and put you in control.
+        </H1>
+        <div>
+          <p>
+            This Privacy Policy is meant to help you understand what information
+            we collect, why we collect it, and how you can update, manage,
+            export, and delete your information.We build a range of services
```
Comment (fragm3) [inline src/components/Privacy/Privacy.react.js]: Space after .

File: src/components/Privacy/Privacy.react.js [hunk 2]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
+            that help millions of people daily to explore and interact with the
+            world in new ways. Our services include:
+            <ul>
+              <li>SUSI AI Smart Speaker</li>
+              <li>Custom Chatbots</li>
+              <li>Personalised Skills</li>
```
Comment (fragm3) [inline src/components/Privacy/Privacy.react.js]: Add SUSI AI Chat Service as well

File: src/components/Privacy/Privacy.react.js [hunk 3]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
+            </ul>
+          </p>
+          <p>
+            You can use our services in a variety of ways to manage your
+            privacy. For example, you can sign up for a SUSI Account if you want
+            to create and manage content like skills or chatbots, or have an
+            extraordinary experience of SUSI AI Smart Speaker.And you can use
```
Comment (fragm3) [inline src/components/Privacy/Privacy.react.js]: Space after .

Comment (fragm3) [reply src/components/Privacy/Privacy.react.js]: Replace `And you` by `You can also`

File: src/components/Privacy/Privacy.react.js [hunk 4]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
+            many SUSI AI services when you’re signed out or without creating an
+            account at all, like chatting with SUSI on web or app.And across our
```
Comment (fragm3) [inline src/components/Privacy/Privacy.react.js]: chat, Space after .

File: src/components/Privacy/Privacy.react.js [hunk 5]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
   return (
     <div>
       <Header title="Privacy" subtitle="Privacy Policy for SUSI" />
```
Comment (akshatnitd) [inline src/components/Privacy/Privacy.react.js]: SUSI.AI

File: src/components/Privacy/Privacy.react.js [hunk 6]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
-              For information about how to contact SUSI, please visit our
-              contact page.
-              <br />
-              <br />
-            </p>
-          </div>
+      <HomeDiv>
+        <H1>
+          When you use our services, you’re trusting us with your information.
+          We understand this is a big responsibility and work hard to protect
```
Comment (akshatnitd) [inline src/components/Privacy/Privacy.react.js]: `and we work hard ...`

File: src/components/Privacy/Privacy.react.js (hunk 2 above)
Comment (akshatnitd) [reply src/components/Privacy/Privacy.react.js]: ^ `SUSI.AI`

File: src/components/Privacy/Privacy.react.js [hunk 7]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
+            world in new ways. Our services include:
+            <ul>
+              <li>SUSI AI Smart Speaker</li>
+              <li>Custom Chatbots</li>
+              <li>Personalised Skills</li>
+            </ul>
+          </p>
+          <p>
+            You can use our services in a variety of ways to manage your
+            privacy. For example, you can sign up for a SUSI Account if you want
```
Comment (akshatnitd) [inline src/components/Privacy/Privacy.react.js]: `SUSI.AI`
Replace SUSI with SUSI.AI at all places

File: src/components/Privacy/Privacy.react.js [hunk 8]
```diff
@@ -14,283 +25,135 @@ const Privacy = props => {
...
+          </p>
+          <h2>Things you create or provide to us</h2>
+          <p>
+            When you create a SUSI AI Account, you provide us with personal
+            information that includes your name and a password. You can also
+            choose to add a phone number, Smart Speaker information or upload
+            photo to your account.
+          </p>
+          <p>
+            We also collect the content you create,upload or receive from others
```
Comment (akshatnitd) [inline src/components/Privacy/Privacy.react.js]: `, `

**Comment by harsh-jindal** [comment]
@fragm3 @akshatnitd Made the changes.

**Pull request merged**